front/app/containers/EventsShowPage/components/MetadataInformation/EventDateStylized/index.tsx

```diff
@@ -31,27 +33,32 @@ const EventDateStylized = ({ event }: Props) => {
       justifyContent="center"
       id="e2e-event-date-stylized"
     >
-      <SingleDateStylized
-        day={startAtMoment.format('DD')}
-        month={startDateMonth}
-        time={
-          isEventMultipleDays
-            ? `${startAtMoment.format('LT')}`
-            : oneDayEventTime
-        }
-      />
+      <ScreenReadableEventDate event={event} />
+      <div aria-hidden="true">
```

Comment:
A small comment to link `aria-hidden` and `ScreenReadableEventDate` could be helpful in the future.